omg/entities/projectile.py

```diff
@@ -38,10 +38,10 @@ def create(cls, init_px: float, init_py: float, angle: float) -> Projectile:
         pass
 
 
-class FireballFactory(ProjectileFactory):
+class FireElementFactory(ProjectileFactory):
```

Comment:
in your current design this does not create a projectile anymore (which means it is not a ProjectileFactory) and you only used their class name (overkill to create a class if you only need a string).